app/pulp/app/viewsets/repository.py

```diff
@@ -124,6 +124,16 @@ class PublisherViewSet(NamedModelViewSet):
     queryset = Publisher.objects.all()
     filter_class = PublisherFilter
 
+    def destroy(self, request, pk):
```

Comment:
i would not be in favour of pk=None not to create confusion, i'd go with just 'pk'